### ISSUE TITLE
Multer is not processing multipart/form-data

### ISSUE THREAD

**Issue by the-yadu** [fox1t/fastify-multer#6]
I am using postman to upload a file to the route using fastify-multer.
It throws the below error.

`{
    "statusCode": 415,
    "code": "FST_ERR_CTP_INVALID_MEDIA_TYPE",
    "error": "Unsupported Media Type",
    "message": "FST_ERR_CTP_INVALID_MEDIA_TYPE: Unsupported Media Type: multipart/form-data; boundary=--------------------------566739000861921958655834"
}`

**Comment by fox1t** [comment]
HI. Can you please provide a reproduction repository here on GitHub so I can start further investigation using your setup?

**Comment by the-yadu** [comment]
Hmmm... Interesting
As asked by you, I started making a reproduction repository with single file server.js and a single route to upload the file.
<img width="620" alt="Screenshot 2020-04-02 at 2 28 33 PM" src="https://user-images.githubusercontent.com/7404181/78230086-7d36c000-74ee-11ea-9d98-6a677725ab90.png">

and I uploaded it without any issue. Strange. This got me thinking to do the same for my project.

`I use fastify-autoloader to load the routes dynamically.`

Then, I tried replicating the same in my project, I took the upload route and got it inside server.js file in root directly and made a route directly with server object. 
*IT WORKED*

Guess there is some issue with autoloader combining with multer.
